components/new_channel_modal/new_channel_modal.jsx

```diff
@@ -165,76 +185,91 @@ export default class NewChannelModal extends React.PureComponent {
             serverError = <div className='form-group has-error'><div className='col-sm-12'><p className='input__help error'>{this.props.serverError}</p></div></div>;
         }
 
-        const createPublicChannelLink = (
-            <button
-                className='style--none color--link'
-                onClick={this.props.onTypeSwitched}
-            >
+        let inputPrefixId = '';
```

Comment:
I don't really understand the purpose of this or why we care about focusing the one textbox when the modal is closed, but since that's in the old version, it's fine to leave around for now.